Improve README.md to read for beginners

shintarou-hiraiwa-d opened the pull request:
README.md を初学者にもわかる内容に変更しました。
よろしくお願いします。

shintarou-hiraiwa-d committed:
Improve README.md to read for beginners

shintarou-hiraiwa-d committed:
Fix some texts

*サーバーを起動する の文言を変更
*本来 push 後に使用するべきではないが、学習の一環と考え、rebase
を行っている

yoshiyuki-miyagi-d commented:
Commit comment で "Fix some texts" などの意図を汲み取れないコメントをするのは止めましょう。
文として履歴が残るので、あとで見た時に意味があるコミットなのかどうか判断できないです。

git でこういうときには便利な機能があります。
「git のコメントをまとめる」とか
https://qiita.com/tsuuuuu_san/items/f708a9f7ea8ab8eb6945
「git commit --amend」を使うと、前の commit に含める動きをするのでそういうのを使うと良いです。

yoshiyuki-miyagi-d commented on README.md:
```diff
@@ -1,11 +1,29 @@
+`node server.js`  
+
+### 正常に動作しているか確認する
+ブラウザで localhost:80/hello-form.html を開きます。  
```
> localhost:80/hello-form.html

正確には、 http://localhost:80/hello-form.html と書いたほうが良い

shintarou-hiraiwa-d replied:
ご指摘いただいたように変更しました。

yoshiyuki-miyagi-d commented on README.md:
```diff
@@ -1,11 +1,29 @@
+コマンドラインを起動して、Repository を clone したいディレクトリへ移動します。  
+`cd cloneしたいディレクトリ`  
+以下のコマンドを入力し、clone します。  
+`git clone https://github.com/shintarou-hiraiwa-d/node-form-practice.git`  
```
```
$ git clone https://github.com/shintarou-hiraiwa-d/node-form-practice.git
```

yoshiyuki-miyagi-d replied:
markdown の ` を3つつなげたコードブロックにしたほうがいい

shintarou-hiraiwa-d replied:
各コマンドの先頭に $ を追加しました。
インラインコードからコードブロックに変更しました。
ご指摘ありがとうございます。

shintarou-hiraiwa-d committed:
レビューで指摘された部分の修正

*localhost:80/hello-form.html の先頭にhttp:// を追加
*各コマンドの先頭に $ を追加
*コマンドを書くためにバッククォートを1つ使っていたが3つ使いコードブロックに変更。

yoshiyuki-miyagi-d commented:
commit message は markdown のフォーマットにちゃんと則らないと正確に直してくれないのですよ

```
*localhost:80/hello-form.html の先頭にhttp:// を追加
*各コマンドの先頭に $ を追加
*コマンドを書くためにバッククォートを1つ使っていたが3つ使いコードブロックに変更。
```

↓

```
* localhost:80/hello-form.html の先頭にhttp:// を追加
* 各コマンドの先頭に $ を追加
* コマンドを書くためにバッククォートを1つ使っていたが3つ使いコードブロックに変更。
```

yoshiyuki-miyagi-d commented:
> Fix some texts
> レビューで指摘された部分の修正

case by case になってしまうが、commit の最初はちゃんと内容を示せるような文にしましょう。

shintarou-hiraiwa-d commented:
Fix some texts
↓
サーバーを起動する 部分の修正


レビューで指摘された部分の修正
↓
hello-form.html のURLを正確に記述、また各コマンド部分の修正

この方が良いでしょうか。
